example-app/src/main/AndroidManifest.xml

```diff
@@ -15,16 +15,21 @@
             android:value="10000000-ffff-ffff-ffff-000000000001" />
 
         <activity
-            android:name=".MainActivity"
-            android:exported="true"
-            android:configChanges="orientation|screenSize|screenLayout|keyboardHidden">
+            android:name=".StartActivity"
+            android:exported="true">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
 
+        <activity
+            android:name=".MainActivity"
+            android:exported="true"
+            android:configChanges="orientation|screenSize|screenLayout|keyboardHidden">
+        </activity>
```

Comment:
What's this 2nd activity for? @CAMOBAP

Reply:
To actually test leak scenario, it impossible to test with a single activity